### ISSUE TITLE
Cannot select Shelly Sub - Plugin in PSU Control for sensing/switching

### ISSUE THREAD

**Issue by patrick-nelissen** [edekeijzer/OctoPrint-PSUControl-Shelly#5]
Installed PSU Control plugin and PSU Control - Shelly plugin, but when I select "Plugin" in the Switching and Sensing method, I cannot select the Shelly switching/sensing plugin. "Choose.." is grayed out and "Get More" brings me back to the page I already installed the Shelly plugin from.

![InstalledPlugins](https://user-images.githubusercontent.com/6655482/120954837-e7f01680-c704-11eb-8d3f-e5bd4d91c9d6.png)
![CannotSelectShellyPlugin](https://user-images.githubusercontent.com/6655482/120954833-e6265300-c704-11eb-9383-77a7c37c4bef.png)

**Comment by edekeijzer** [comment]
@patrick-nelissen did you restart Octoprint after installation? I'm not sure what the lightning bolt behind the plugin name is indicating.

**Comment by patrick-nelissen** [comment]
@edekeijzer  Yes I did. I believe the "lightning bolt" indicates that the "lightning bolt" button on the main page is controlled by this sub-plug-in. That button allows you to "switch" the PSU control on/off.

**Comment by edekeijzer** [comment]
> @edekeijzer Yes I did. I believe the "lightning bolt" indicates that the "lightning bolt" button on the main page is controlled by this sub-plug-in. That button allows you to "switch" the PSU control on/off.

I'm quite sure that's not it, as I do not have this symbol in my test environment where the Shelly plugin is selected as well. What does it say when you hover the mouse over it?

**Comment by patrick-nelissen** [comment]
@edekeijzer That explains it. There appearsto be a compatibility issue.
![Incompatible](https://user-images.githubusercontent.com/6655482/121200964-fdf6f780-c828-11eb-873e-acf0fa6372b3.png)

I checked my my versions: [octoprint-systeminfo-20210608151518.zip](https://github.com/edekeijzer/OctoPrint-PSUControl-Shelly/files/6617269/octoprint-systeminfo-20210608151518.zip)

Looks like I am still running Phyton 2.7.16...

I upgraded to Phyton 3, which resolved the compatibility issue. Will keep you posted if I run into any other problems.
